### DISCUSSION TITLE
REMINDERS!

### DISCUSSION THREAD

**Discussion by xylabolos** [xylabolos/17F-Madis#6] [Announcements]
- **THURDAY MEETING** (9 PM)
meeting will include finalizing and testing of codes
- gmeet link : meet.google.com/orp-avxv-fcs
- **FRIDAY VIDEO SHOOTING** (WHOLE DAY)

**Comment by rofelmaetayoto** [comment]
Noted 👍

**Reply by rofelmaetayoto** [reply]
heheehehe
